lib/fluent/plugin/filter_kubernetes_metadata.rb

```diff
@@ -58,6 +58,8 @@ class KubernetesMetadataFilter < Fluent::Plugin::Filter
     config_param :secret_dir, :string, default: '/var/run/secrets/kubernetes.io/serviceaccount'
     config_param :de_dot, :bool, default: true
     config_param :de_dot_separator, :string, default: '_'
+    config_param :de_slash, :bool, default: true
```

Comment:
This should default to `false` to ensure backwards compatibility

Reply:
✅